spec/lib/political_edition_identifier_spec.rb

```diff
@@ -0,0 +1,77 @@
+# frozen_string_literal: true
+
+RSpec.describe PoliticalEditionIdentifier do
+  describe ".policical_organisation_ids" do
+    it "returns an array of content_ids" do
+      uuid_regex = /^[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}$/
+      expect(PoliticalEditionIdentifier.political_organisation_ids)
+        .to all match(uuid_regex)
```

Comment:
May be worth asserting the length of the `political_organisation_ids` array here - we're already hardcoding the YAML file of orgs, so may as well hard code the number of orgs we expect to see.

My concern is that `political_organisation_ids` could return `[]` and all the tests would still currently pass.

Reply:
I don't want to assert the length as I don't want the test to fail if the length of the list changes. However you raise a good point about the empty approach, so I'll add an assertion that it's not empty.